assets/js/payment-methods-demo/payment-methods/index.js

```diff
@@ -1,41 +1,25 @@
 /**
- * Internal dependencies
+ * External dependencies
  */
-import { paypalSvg } from './paypal';
-import { ccSvg } from './cc';
-
-const PaypalActivePaymentMethod = () => {
-	return (
-		<div>
-			<p>This is where paypal payment method stuff would be.</p>
-		</div>
-	);
-};
+import { __ } from '@wordpress/i18n';
 
-const CreditCardActivePaymentMethod = () => {
-	return (
-		<div>
-			<p>This is where cc payment method stuff would be.</p>
-		</div>
-	);
-};
+/**
+ * Internal dependencies
+ */
+import { PAYMENT_METHOD_NAME, StripeCreditCard } from './stripe';
+import { stripePromise } from '../stripe-utils';
 
-export const paypalPaymentMethod = {
-	id: 'paypal',
-	label: <img src={ paypalSvg } alt="" />,
-	stepContent: <div>Billing steps</div>,
-	activeContent: <PaypalActivePaymentMethod />,
-	edit: <PaypalActivePaymentMethod />,
-	canMakePayment: Promise.resolve( true ),
-	ariaLabel: 'paypal payment method',
-};
+const EditPlaceHolder = () => <div>TODO: Card edit preview soon...</div>;
 
-export const ccPaymentMethod = {
-	id: 'cc',
-	label: <img src={ ccSvg } alt="" />,
-	stepContent: null,
-	activeContent: <CreditCardActivePaymentMethod />,
-	edit: <CreditCardActivePaymentMethod />,
-	canMakePayment: Promise.resolve( true ),
-	ariaLabel: 'credit-card-payment-method',
+export const stripeCcPaymentMethod = {
+	id: PAYMENT_METHOD_NAME,
+	label: (
+		<strong>
```

Comment:
Maybe this (strong) should be moved into where this method is rendered so gateways are not responsible.

Reply:
The `label` property for the config can receive any React node. This allows for gateways that might pass in an image instead of text (i.e. Paypal) for the label.  So while I agree somewhat with what you're saying here, I think we have to decide first about how restrictive we want to be for payment methods registering nodes for the tab label.